src/operations/command.ts

```diff
@@ -169,6 +166,6 @@ export abstract class CommandOperation<T> extends AbstractOperation<T> {
       }
     }
 
-    server.command(this.ns, cmd, { fullResult: !!this.fullResponse, ...options }, callback);
+    server.command(this.ns, cmd, { ...options }, callback);
```

Comment:
Is there a reason to spread this anymore?

Reply:
nope